plugins/buildx.installer

```diff
@@ -6,7 +6,7 @@ source "$(dirname "$0")/.common"
 PKG=github.com/docker/buildx
 GOPATH=$(go env GOPATH)
 REPO=https://${PKG}.git
-: "${BUILDX_COMMIT=v0.5.1}"
+: "${BUILDX_COMMIT=v0.6.0}"
```

Comment:
```suggestion
: "${BUILDX_COMMIT=v0.6.1}"
```

Ref #565

Reply:
I'll cherry pick the other commit as well 👍